pxr/imaging/hd/camera.cpp

```diff
@@ -255,6 +255,41 @@ HdCamera::Sync(HdSceneDelegate * sceneDelegate,
             _exposure = vExposure.Get<float>();
         }
 
+        const VtValue vExposureCompensation =
```

Comment:
We should add initialization for all of these member variables in the HdCamera constructor (following the pattern for _exposure).